packages/next/README.md

```diff
@@ -2316,6 +2316,22 @@ To learn more about TypeScript checkout its [documentation](https://www.typescri
 > **Note**: Next.js does not enable TypeScript's `strict` mode by default.
 > When you feel comfortable with TypeScript, you may turn this option on in your `tsconfig.json`.
 
+> **Note**: By default, Next.js reports TypeScript errors during development for pages you are actively working on.
+> TypeScript errors for inactive pages do not block the development process.
+> Trying to run `next build` for an app that has TypeScript errors on any page will fail.
+> 
+> If you don't want to leverage this behavior and prefer to do type checks manually, set the following options in your `next.config.js`:
```

Comment:
I'd prefer to split these into two separate sections. 🙏

Reply:
That'd be great! I have a few more hours before going offline till Monday. If we don't sort out all the final amendments by then, I'll be happy for you guys to push directly to my branch and complete the PR if you want 🙌 😉

Reply:
@Timer I'm back and will be happy to address further comments, if any.

Reply:
@kachkaev I'd be fine with landing it and adding the docs afterwards in a separate PR to discuss 👍

Reply:
@timneutkens cool! The PR should be good to go then, unless you or @Timer have further suggestions. Weirdly the CI is failing, but that seems to be the issue with dynamic routing tests rather than what my PR has added.